mender-convert

```diff
@@ -343,21 +363,25 @@ do_make_sdimg_beaglebone() {
   local ret=0
 
   create_device_maps $raw_disk_image raw_disk_mappings
-
-  mount_mender_disk ${mender_disk_mappings[@]}
   mount_raw_disk ${raw_disk_mappings[@]}
 
   log "$step/$total Setting boot partition..."
   ((step++))
-  stage_2_args="$sdimg_boot_dir $embedded_rootfs_dir"
+
+  stage_2_args="$output_dir ${mender_disk_mappings[0]} ${embedded_rootfs_dir}"
   ${tool_dir}/bbb-convert-stage-2.sh ${stage_2_args} || ret=$?
   [[ $ret -ne 0 ]] && { log "Aborting."; return $ret; }
 
+  unmount_partitions $raw_disk_image
```

Comment:
No need to call `unmount_partitions`, `detach_device_maps` called in `do_raw_disk_image_create_partitions()` will do that job.
In fact `unmount_partitions()` method is not used in any other place - can be removed I think.

Reply:
Dropped this